java/web/skeleton/Makefile

```diff
@@ -101,25 +121,79 @@ push-functional: ## Push the Functional Docker image to an image registry
 push-nft: ## Push the NFT Docker image to an image registry
 	docker image push $(REGISTRY)/$(FAST_FEEDBACK_PATH)/$(image_name)-nft:$(image_tag)
 
+.PHONY: push-extended
+push-extended:
+	docker image push $(REGISTRY)/$(EXTENDED_TEST_PATH)/$(image_name)-extended:$(image_tag)
+
 .PHONY: deploy-functional
-deploy-functional: ## Deploy helm chart of the reference app to functional namespace
-	helm upgrade --install $(app_name)-dependencies ./helm-charts/ref-app-dependencies -n $(app_name)-functional --set registry=$(REGISTRY)/$(FAST_FEEDBACK_PATH) --atomic
-	helm upgrade --install $(app_name) ./helm-charts/ref-app -n $(app_name)-functional --set registry=$(REGISTRY)/$(FAST_FEEDBACK_PATH) --set domain=$(BASE_DOMAIN) --set appUrlSuffix="-functional" --set tag=$(image_tag) --atomic
-	helm list -n $(app_name)-functional ## list installed charts in the given tenant namespace
+deploy-functional: ## Deploy helm chart of the app to functional namespace
+	helm upgrade --install $(app_name) helm-charts/app -n $(tenant_name)-functional \
+		--set registry=$(REGISTRY)/$(FAST_FEEDBACK_PATH) \
```

Comment:
maybe for another task, but having so many sets feels cluttered. I'd move to have a values file per environment. If you agree, we can create a task for it

Reply:
I'll include this point in the existing task